assets/js/components.js

```diff
@@ -3,4 +3,25 @@ $(function() {
    $(this).parent().removeClass('hidden')
     .siblings().addClass('hidden');
   });
+
+
+  var footerAccordion = function() {
```

Comment:
This is introducing globals variables, we should file and issue to change to a better module scheme.